src/video_core/renderer_opengl/gl_shader_manager.cpp

```diff
@@ -267,7 +267,7 @@ class ShaderDoubleCache {
             auto [iter, new_shader] = shader_cache.emplace(program, OGLShaderStage{separable});
             OGLShaderStage& cached_shader = iter->second;
             if (new_shader) {
-                result->code = program;
+                result.code = program;
```

Comment:
```suggestion
                result.emplace();
                result->code = program;
```
Doing this instead would most likely solve the crash.